src/main/java/org/folio/service/orders/OrderRolloverService.java

```diff
@@ -144,15 +141,17 @@ private Future<Void> handleOrderRolloverError(Throwable t, LedgerFiscalYearRollo
   }
 
   private Future<Void> rolloverOrdersByFundIds(List<String> ledgerFundIds, LedgerFiscalYearRollover ledgerFYRollover, String systemCurrency, RequestContext requestContext) {
-    if (CollectionUtils.isEmpty(ledgerFundIds)) return Future.succeededFuture();
-
+    if (CollectionUtils.isEmpty(ledgerFundIds)) {
+      return Future.succeededFuture();
+    }
     var fundIdChunks = Lists.partition(ledgerFundIds, MAX_IDS_FOR_GET_RQ_15);
-
-    return requestContext.getContext().<List<Future<Void>>>executeBlocking(promise -> {
-        List<Future<Void>> futures = new ArrayList<>();
+    var ctx =  requestContext.getContext();
+    return ctx
+      .<List<Future<Void>>>executeBlocking(promise -> {
+        var futures = new ArrayList<Future<Void>>();
         // process fundId chunks one by one
-        Semaphore semaphore = new Semaphore(1, requestContext.getContext().owner());
-        for (List<String> fundIds : fundIdChunks) {
+        var semaphore = new Semaphore(1, requestContext.getContext().owner());
+        for (var fundIds : fundIdChunks) {
```

Comment:
do you think that this formatting style is better then previous? for me it is the same, but let it be

Reply:
I just saw that there are 2 styles here in this method i.e. reference types & var together, I think everything inside should be either one or the other to make it more readable